### PR TITLE
Update Documentation and Codecov

### PR DESCRIPTION
**Pull request by andrewrosemberg**
- Updates documentation
- Adds link to documentation in Readme
- Fixes Codecov to ignore plots

### CODE REVIEW

File: src/plot.jl [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     xguide --> "Strategy"
     yguide --> "round"
+    label --> "score"
```
Comment (Ianlmgoddard) [inline src/plot.jl]: I think you want this 
```suggestion
    colorbar_title --> "score"
```

**Comment by andrewrosemberg** [comment]
Ready for final review. Docs are building fine.

File: src/plot.jl [hunk 2]
```diff
@@ -8,12 +8,13 @@
...
     xguide --> "Strategy"
     yguide --> "round"
+    colorbar_title --> "score"
     seriestype --> :heatmap
     seriescolor --> :RdYlGn_10
 
     ystrings = reverse(string.(rounds))
 
-    xlabels = (1:size(scores, 2), strategies)
+    xlabels = (1:size(scores, 2), last.(split.(string.(strategies), ".")))
```
Comment (Ianlmgoddard) [inline src/plot.jl:17]: It works I guess, but it's a bit gross.  Did you figure out what the issue was?

**Review by Ianlmgoddard — APPROVED** [review]

File: src/plot.jl (hunk 2 above)
Comment (andrewrosemberg) [reply src/plot.jl:17]: Not quite, but I will continue to look into this after the MR. This also happens if I don't export Devil or RandomStrategy.

**Pull request merged**